src/quoridor/QuoridorBoard.cs

```diff
@@ -0,0 +1,45 @@
+﻿using QuoridorEngine.src.solver;
+
+namespace QuoridorEngine.src.quoridor
+{
+    /// <summary>
+    /// Class representing quoridor board
+    /// </summary>
+    internal class QuoridorBoard
+    {
+        List<(int, int)> usedWalls;
+        private int width;
+        private int height;
+
+        public int Width
```

Comment:
The quoridor boards are square by definition so using two different variables is redundant. Maybe use only one variable names Dimension instead?